docs/src/pages/index.js

```diff
@@ -325,12 +325,11 @@ export default function IndexPage() {
             />
             <div className="container01">
               <div className="container02">
-                <h1 className="text01">
-                  <span>Get WhatsApp</span>
-                  <br></br>
-                  <span>notification when your</span>
-                </h1>
-                <h1 className="text05">web app is down</h1>
+                <pre className="text01">
+                  Get WhatsApp {'\n'}
+                  notification when {'\n'}
```

Comment:
please don't use new line `\n` here. It's better to change the width container instead so that the text is not polluted with unnecessary new lines.